spark-cassandra-connector-java/src/main/java/com/datastax/spark/connector/japi/RDDAndDStreamCommonJavaFunctions.java

```diff
@@ -177,7 +177,7 @@ public WriterBuilder withBatchSize(BatchSize batchSize) {
                 return withWriteConf(
                     new WriteConf(batchSize, writeConf.batchGroupingBufferSize(), writeConf.batchGroupingKey(),
                         writeConf.consistencyLevel(), writeConf.parallelismLevel(), writeConf.throughputMiBPS(),
-                        writeConf.ttl(), writeConf.timestamp()));
+                        writeConf.ttl(), writeConf.timestamp(), writeConf.taskMetricsEnabled()));
```

Comment:
Yeah, I'm gonna work on cumulative updates and fixed in Java API, so I can address this